html/mycakeapp/src/Controller/AuctionController.php

```diff
@@ -86,17 +86,33 @@ public function add()
 		$biditem = $this->Biditems->newEntity();
 		// POST送信時の処理
 		if ($this->request->is('post')) {
-			// $biditemにフォームの送信内容を反映
-			$biditem = $this->Biditems->patchEntity($biditem, $this->request->getData());
-			// $biditemを保存する
-			if ($this->Biditems->save($biditem)) {
-				// 成功時のメッセージ
-				$this->Flash->success(__('保存しました。'));
-				// トップページ（index）に移動
-				return $this->redirect(['action' => 'index']);
+			// アップロードを許可する拡張子を用意
+			$extensions = array('jpg', 'jpeg', 'png', 'gif', 'JPG', 'JPEG', 'PNG', 'GIF');
+			// アップロードファイルの拡張子取得
+			$getExtension = pathinfo($_FILES['image']['name'] , PATHINFO_EXTENSION);
```

Comment:
```suggestion
			$fileExtension = pathinfo($_FILES['image']['name'] , PATHINFO_EXTENSION);
```

多くのプログラミング言語では
* 変数名は名詞
* 関数名は動詞+目的語
になります。
まずはこのルールになっているか念頭に良い名前がないか考えてみましょう。